content/F2015/F15-event-career-panel.md

```diff
@@ -0,0 +1,21 @@
+title: CSC and WiCS Career Panel
+Date: 2015-09-24 17:00-19:00
```

Comment:
The featured post is simply the "most recent" article in the blog. Fatema chose a Sep. 13th `date` for the Orientation post, as though it was written in the future. It will remain the featured post until a more recent post is added to the blog, i.e. one with a post `date` of Sep. 14th. If I were to add a post to the blog with today's date, it would be less recent, so it would not be the featured post, but will still show up in the top 5 since it's more recent than WiCS studies.

I'm not sure if there's a "featured" tag/metadata you can specify to override the date-based default. I should look into that.